src/components/FeatureViews/Registration/RegistrationLanding.js

```diff
@@ -42,26 +55,32 @@ const RegistrationLanding = () => {
     }, []);
 
     const instructorOptions = useMemo(() => Object.values(instructors)
-        .map(({name, user_id}) => ({
+        .map(({ name, user_id }) => ({
             "label": name,
             "value": user_id,
         })), [instructors]);
 
+    const subjectOptions = useMemo(() => distinctObjectArray(Object.values(courses)
+        .map(({ course_id, category }) => ({
+            "label": categories.find(Category => Number(Category.id) === Number(category)).name,
+            "value": category,
+        }))), [courses])
```

Comment:
Why not simply pull categories from redux directly?

Reply:
We need to match categories. Inside redux the category value is an integer that needs to match with the category object